tests/src/test/java/alluxio/testutils/master/MasterTestUtils.java

```diff
@@ -64,6 +69,19 @@ public static MasterRegistry createStandbyFileSystemMasterFromJournal() throws E
     return createFileSystemMasterFromJournal(false, null);
   }
 
+  /**
+   * Creates a new leader {@link FileSystemMaster} from journal along with its dependencies, and
+   * returns the master registry containing that master.
+   *
+   * @return a master registry containing the created {@link FileSystemMaster} master
+   */
+  public static MasterRegistry createSnapshotLeaderFileSystemMasterFromJournal() throws Exception {
```

Comment:
I'm not sure what a "snapshot leader" means. maybe this should say `createLeaderFileSystemMasterFromJournalCopy()`?